### PR TITLE
Portfolio-2nd-project

### PR DESCRIPTION
**Pull request by feyisakedir**
### There is an update in the following files:
- Index.html file committing
- README.md file Updated
- styles.css file is committed

### CODE REVIEW

File: gitignore/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+
```
Comment (joel-kalema) [inline gitignore/.gitignore:2]: - [ ] Good job working on the "Portfolio: setup and mobile first" 😃, but currently, your project doesn't have any `index.html` and `css` files 😸. Kindly make sure you create them and customize your project according to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=rHQ6yd5LRBvNFkFE-0) given, and make sure your website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.).

## See the  [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) 😃

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @feyisakedir 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark:  
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by feyisakedir** [comment]
you are my code reviewer on the portfolio project. you gave me this
required changes "

   -  Check the comments under the review.   but I did not understand it
   make it clear sir.

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @feyisakedir 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] [OPTIONAL] It could be better if you remove your `index.html` and `css` files from `git ignore` falder😸.


- [ ] Good job working on the "Portfolio: setup and mobile first" 😃, but kindly, make sure your project looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.).




> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-05-03 1| |Screenshot 2023-05-03 101919| |
Screenshot 2023-05-03 102842

- [ ] For best practice, Kindly make sure you follow this syntax:
````
     <header>
       <a href="#">My Logo<a/>
        <nav>
            <img class="menu"src="menu.png" alt="menu">
        </nav>       
      </header>
       <main>
             <section>
              ....
             </section>
       </main>
```` 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @feyisakedir      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights 
- Right linter files✔️  
- Appropriate PR  ✔️  
  
# Required Changes  🌵 

- [ ] I understand this is your last review, and in as much as I would have loved to approve you to see you move to the next project, you have not made any changes from the ones suggested by the last reviewer .
- The idea of this review process is to help you build these skills which are paramount to be a professional developer

- [ ] I advice you to take the suggestion in good fate, and do make the changes based on the skills you've gained from the curriculum available. This [Resource](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) on flexbox will help you going forward to structure the content which I noticed was the major issue you had.

## As suggested by the last reviewer:
- This change is necessary to test your skills in the semantics of `HTML`
image

## Changes suggested not effected:
- Your design is not still structured as seen below compared to the Figma design. The idea is to see how you can replicate a given design. Kindly check the link i attached above on flexbox to carry this on
image

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**